### PR TITLE
feat(helm): add support for providing additional volumes and adding init containers

### LINKED ISSUE #454: Support additional Volumes and Init containers for helm install
**Describe the problem/challenge you have**
Need to be able to initialize some nodes prior to plugin execution

**Describe the solution you'd like**
Add support to the deploy yaml to allow additional volumes and init containers to be specified via values.yaml

**Anything else you would like to add:**

### PR DESCRIPTION
**Pull request by jnels124**
## Pull Request template

Please, go through these steps before you submit a PR.

**Why is this PR required? What issue does it fix?**:
Adds support for additional volumes and init containers
**What this PR does?**:
Updates deployment yaml to support volume and init container config via values.yaml

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:
deployed helm chart with various init containers and additional volumes. 
Verified init containers executed correctly.

**Any additional information for your reviewer?** :

**Checklist:**
- [x] Fixes #454 
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [x] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

**Comment by niladrih** [comment]
Why is this change required?

**Comment by jnels124** [comment]
@niladrih This is required for our usecase because we would like to automatically bootstrap the nodes with the setup required for zfs. [Currently, we are doing this via a daemon set](https://github.com/hashgraph/hedera-mirror-node/tree/main/charts/hedera-mirror-common/templates/zfs) that runs an init container and then a pause container(simply because a container is required). It would be simpler if we could just specify an init container to run within the node and controller deployments.

File: deploy/helm/charts/templates/zfs-contoller.yaml [hunk 1]
```diff
@@ -38,6 +38,13 @@ spec:
       priorityClassName: {{ template "zfslocalpv.zfsController.priorityClassName" . }}
 {{- end }}
       serviceAccount: {{ .Values.serviceAccount.zfsController.name }}
+{{- if .Values.zfsController.initContainers }}
```
Comment (steven-sheehy) [inline deploy/helm/charts/templates/zfs-contoller.yaml:41]: Should add initContainers and additionalVolumes to values.yaml with `[]` default.

**Comment by niladrih** [comment]
These changes have been included in https://github.com/openebs/zfs-localpv/pull/461, closing this one. This is available in the released version v2.3.0.

**Comment by jnels124** [comment]
These changes were added to 2.3

**Pull request closed without merging**